config/txgh.yml

```diff
@@ -1,15 +0,0 @@
-txgh:
-    github:
-        repos:
-            matthewjackowski/txgh-test-resources:
-                api_username: <%= ENV['GITHUB_USERNAME']  %>
-                api_token: <%= ENV['GITHUB_TOKEN'] %>
-                push_source_to: <%= ENV['GITHUB_PUSH_SOURCE_TO'] %>
-                branch: <%= ENV['GITHUB_BRANCH'] %>
-    transifex:
-        projects:
-            txgh-test-1:
-                tx_config: <%= ENV['TX_CONFIG_PATH'] %>
-                api_username: <%= ENV['TX_USERNAME'] %>
-                api_password: <%= ENV['TX_PASSWORD'] %>
-                push_translations_to: <%= ENV['TX_PUSH_TRANSLATIONS_TO'] %>
```

Comment:
In supporting this project I find that I end up spending more time supporting Heroku toolbelt or Ubuntu (AWS)...ie the platform instead of the app.  And this bums me out.

So 1 approach I was thinking was something along the lines of this (sample code below).  A simple command-line 'UI' we could use to set the ENV in order to get up and running and then have an separate export/save per platform Du jour (ie to Heroku via config_env gem AWS via just YAML, etc).
def ask
      VARIABLES.each do |var|
        print "Please enter your #{var}:  "
        value = gets.strip
        redo unless value =~ /\S/
        ENV[var] = value
      end
      save
    end